elements/pfe-button/package.json

```diff
@@ -51,8 +51,6 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@patternfly/pfe-sass": "1.0.0-prerelease.40",
-    "@patternfly/pfelement": "1.0.0-prerelease.39",
-    "gh-pages": "^2.2.0"
+    "@patternfly/pfelement": "^1.0.0"
```

Comment:
Can you move pfe-sass to a devDependency?  It's still a requirement for the component.

Reply:
This is done. Thanks.